server/src/test/java/org/spine3/server/projection/ProjectionRepositoryShould.java

```diff
@@ -477,10 +474,37 @@ private ManualCatchupProjectionRepository repoWithManualCatchup() {
         return repo;
     }
 
+    @CanIgnoreReturnValue
+    private Collection<ProjectId> setUpEvents(BoundedContext boundedContext, int eventCount) {
+        // Set up bounded context
+        final EventStore eventStore = boundedContext.getEventBus()
+                                                    .getEventStore();
+        final Collection<ProjectId> ids = new LinkedList<>();
+        for (int i = 0; i < eventCount; i++) {
+            final ProjectId projectId = ProjectId.newBuilder()
+                                                 .setId(valueOf(i))
+                                                 .build();
+            final Message eventMessage = ProjectCreated.newBuilder()
+                                                       .setProjectId(projectId)
+                                                       .build();
+            final Event event = createEvent(pack(projectId), eventMessage);
+            appendEvent(eventStore, event);
+        }
+        return ids;
+    }
+
     private StorageFactory storageFactory() {
         return StorageFactorySwitch.get(boundedContext.isMultitenant());
     }
 
+    private static void await(long millis) {
```

Comment:
I would avoid this method as much as possible.

Besides, `AssertionError` is not used properly, as it's not what you, probably, think what it is. Check out its documentation and pay special attention to its package.